pkg/network/tracer/tracer.go

```diff
@@ -242,6 +243,14 @@ func newConntracker(cfg *config.Config, bpfTelemetry *nettelemetry.EBPFTelemetry
 
 	var c netlink.Conntracker
 	var err error
+
+	cmd := exec.Command("modprobe", "nf_conntrack_netlink")
+	err = cmd.Run()
+
+	if err != nil {
+		log.Warnf("error enabling probe nf_conntrack_netlink: %s", err)
+	}
+
 	if c, err = NewEBPFConntracker(cfg, bpfTelemetry); err == nil {
```

Comment:
We shouldn't continue on to this if the module isn't loaded, right?

Reply:
I suppose that's fair... I guess I thought it couldn't hurt to try again 🤷